src/test/scala/com/elevator/ElevatorSystemSpec.scala

```diff
@@ -4,11 +4,11 @@ import org.specs2.Specification
 import org.specs2.concurrent.ExecutionEnv
 import org.specs2.specification.AroundTimeout
 
+import scalaz.zio.{ZIO, IO, Schedule, DefaultRuntime}
+import scalaz.zio.duration._
 
-import scala.concurrent.duration._
-import scalaz.zio.{IO, RTS, Schedule}
+class ElevatorSystemSpec(implicit ee: ExecutionEnv) extends Specification with AroundTimeout {
```

Comment:
Consider to extend the DefaultRuntime so we can use unsafeRun all the time in this test.